x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -342,6 +342,19 @@ export function updateEditShape(shapeToDraw: DRAW_SHAPE | null) {
   };
 }
 
+export function setSelectedLayerToEditLayer() {
```

Comment:
The method name is backwards. You are not seting selected layer Id to edit layer. The method is setting edit layer to the selected layer id. How about changing to setEditLayerToSelectedLayer